.github/workflows/build-container.yaml

```diff
@@ -20,6 +20,7 @@ on:
 
 jobs:
   build:
+    if: github.repository == 'mfem/mfem' # Don't run in forks
```

Comment:
If this works, then LGTM! I usually have an if like this, but targeted for specific events, e.g.:

```
github.event.pull_request.head.repo.full_name
```
but this particular workflow has 4 events, all of which should be on mfem/mfem, so theoretically it should work (but it would be good to confirm it).

Reply:
I tested it [here](https://github.com/pazner/mfem/actions/runs/2517178635), it looks like it works

Reply:
We use 

```
    if: |
      (github.event_name == 'push' ||
       github.event.pull_request.head.repo.full_name != github.repository)
```

for the same purpose in the [`repo-check` workflow](https://github.com/mfem/mfem/blob/master/.github/workflows/repo-check.yml#L32-L34).

I thought that there was a reason to do it this way, but I can't remember now and I could be wrong.

If this simpler version works after merging in `master`, we should also update `repo-check` to use it.

Reply:
My only concern with this check is that it might only pertain to the pull request event? So it might work to disable the job in even the right repository but different trigger.

Reply:
It looks like the `github.repository` value is documented [here](https://docs.github.com/en/actions/learn-github-actions/contexts#github-context). As @vsoch mentioned, this probably makes more sense because this action is not limited to PRs.

Reply:
Didn't notice that. All good then